neo.UnitTests/neo.UnitTests.csproj

```diff
@@ -32,4 +32,10 @@
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="./libleveldb.dll">
```

Comment:
this could produce undesired results in different architectures

Reply:
We hope to be able to test Persistence module using libleveldb, so what do we do if don't copy the dll file?

Reply:
we should copy it acording to the architecture, or assume that is copied and install it before in travis

Reply:
This will break on other architectures, but don't think you need it... nuget packages aren't enough?

Reply:
Windows environment needs this file, otherwise unit test will not work properly. Linux environment only needs to install leveldb before travis and the existence of  this file will not affect the operation of UT.  We are still working on travis to fix Mac environment. 

It seems that Nuget package cannot replace libleveldb because leveldb is dynamically loaded at runtime.

Reply:
We should care about travis, at now devs should copy manually for run neo, so for me  is not a problem if you already have it in your path.

Reply:
is for x64 or for x32?

Reply:
It's not a good idea to include `libleveldb.dll` in unit tests. Maybe we can create some mocked persistence classes.

Reply:
Got it, we will try to use mocked data to test these.